lib/galaxy/tool_util/lint.py

```diff
@@ -158,22 +201,23 @@ def warn_messages(self):
     def error_messages(self):
```

Comment:
I think it would be more useful to return here the actual `List[LintMessage]` with all the metadata instead of just the string message. The consumer of this property can then make use of the metadata if required.

Reply:
Done.